frontend/apps/app/components/SessionDetailPage/components/Output/components/ERD/ERD.tsx

```diff
@@ -1,4 +1,5 @@
 import type { Schema } from '@liam-hq/db-structure'
+import { ERDRendererProvider } from '@liam-hq/erd-core/nextjs'
```

Comment:
Each `ERDRenderer` needs to be wrapped by the `ERDRendererProvider` component of the suitable framework.